code/modules/modular_computers/computers/machinery/console_presets.dm

```diff
@@ -44,7 +44,7 @@
 
 // ===== RESEARCH CONSOLE =====
 /obj/machinery/modular_computer/console/preset/research
-	 console_department = "Medbay"
+	 console_department = "Research"
```

Comment:
I noticed this while poking around, it looks like someone copypasted without realising the mistake.